instrumentation/sample/src/test/kotlin/de/mannodermaus/sample/ExampleKotlinTest.kt

```diff
@@ -100,6 +103,12 @@ class ExampleKotlinTest {
         Executable { assertNotNull(value) },
         Executable { assertEquals(15, value.length) })
   }
+  
+  @ParameterizedTest(name = "Upper case for {0}")
+  @MethodSource("getNames")
+  fun parameterizedMethodTest (names: Pair<String, String>) {
+      assertEquals(names.first.toUpperCase(), names.second)
```

Comment:
`names.first.toUpperCase()` produces the actual result, `names.second` is the expected outcome. Please switch the param order here, as `assertEquals()` takes the expected one first!

Reply:
DOne